### PR TITLE
Express Route Cross Connection Swagger changes as per PM requirements

### PR DESCRIPTION
**Pull request by vinayada1**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json [hunk 1]
```diff
@@ -822,6 +809,117 @@
         }
       },
       "description": "Response for ListExpressRouteCrossConnection API service call."
+    },
+    "ExpressRouteCrossConnectionPeeringPropertiesFormat": {
```
Comment (gimotwanMSFT) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json]: Please review and update attributes that are readOnly in this definition

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json [hunk 2]
```diff
@@ -822,6 +809,117 @@
...
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExpressRouteCrossConnectionPeering"
+          },
+          "description": "The peerings in an express route cross connection."
+        },
+        "nextLink": {
+          "type": "string",
```
Comment (gimotwanMSFT) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json]: nextLink should be readOnly

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "parameters": {
     "peeringName": "AzurePrivatePeering",
-    "crossConnectionName": "CrossConnection-6e918437-5077-4ffe-b45f-0b7c0672da4b",
+    "crossConnectionName": "6e918437-5077-4ffe-b45f-0b7c0672da4b",
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json]: Please rename these as something generic like "connectionString", anything resembling actual keys should be avoided here

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json [hunk 4]
```diff
@@ -19,7 +19,7 @@
     "201": {
       "body": {
         "name": "AzurePrivatePeering",
-        "id": "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/CrossConnection-SiliconValley/providers/Microsoft.Network/expressRouteCrossConnections/CrossConnection-6e918437-5077-4ffe-b45f-0b7c0672da4b/peerings/AzurePrivatePeering",
+        "id": "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/CrossConnection-SiliconValley/providers/Microsoft.Network/expressRouteCrossConnections/6e918437-5077-4ffe-b45f-0b7c0672da4b/peerings/AzurePrivatePeering",
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json]: Same comment as above for the `subscriptionId` here

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json (hunk 1 above)
Comment (dsgouda) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json]: A better name would be `ExpressRouteCrossConnectionPeeringProperties`

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json [hunk 5]
```diff
@@ -822,6 +809,120 @@
...
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Peering in an ExpressRoute Cross Connection resource."
+    },
+    "ExpressRouteCrossConnectionPeeringListResult": {
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/expressRouteCrossConnection.json]: A better name would be `ExpressRouteCrossConnectionPeeringList`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Apart from a few minor comments, looks okay. Need to check linter/model validator issues

**Comment by dsgouda** [comment]
As an FYI, existing and newer linter validations should be addressed for newer API versions per our offline conversations.

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json (hunk 3 above)
Comment (gimotwanMSFT) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json (hunk 4 above)
Comment (gimotwanMSFT) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/examples/ExpressRouteCrossConnectionBgpPeeringCreate.json]: Fixed

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**